tests/jetty-test-session-common/src/main/java/org/eclipse/jetty/session/test/tools/RemoteInfinispanTestSupport.java

```diff
@@ -50,35 +49,39 @@ public class RemoteInfinispanTestSupport
     private static final Logger LOG = LoggerFactory.getLogger(RemoteInfinispanTestSupport.class);
     public RemoteCache<String, InfinispanSessionData> _cache;
     private final String _name;
-    public static RemoteCacheManager _manager;
     private static final Logger INFINISPAN_LOG =
             LoggerFactory.getLogger("org.eclipse.jetty.server.session.remote.infinispanLogs");
 
-    private static final String INFINISPAN_VERSION = System.getProperty("infinispan.docker.image.version", "14.0.25.Final");
+    private static final String INFINISPAN_VERSION = System.getProperty("infinispan.docker.image.version", "15.0.9.Final");
     private static final String IMAGE_NAME = System.getProperty("infinispan.docker.image.name", "infinispan/server") +
             ":" + INFINISPAN_VERSION;
 
-    private static final GenericContainer<?> infinispan = new GenericContainer<>(IMAGE_NAME);
-
-    static
+    private static final class Holder
     {
-        infinispan.withEnv("USER", "theuser")
-                .withEnv("PASS", "foobar")
-                .withEnv("MGMT_USER", "admin")
-                .withEnv("MGMT_PASS", "admin")
-                .withEnv("CONFIG_PATH", "/user-config/config.yaml")
-                .waitingFor(Wait.forListeningPorts(11222))
-                .withExposedPorts(4712, 4713, 8088, 8089, 8443, 9990, 9993, 11211, 11222, 11223, 11224)
-                .withLogConsumer(new Slf4jLogConsumer(INFINISPAN_LOG))
-                .withClasspathResourceMapping("/config.yaml", "/user-config/config.yaml", BindMode.READ_ONLY)
-                .start();
-
-        // setup instance
+        private static final Holder INSTANCE = new Holder();
+
+        private final RemoteCacheManager manager;
+
+        private Holder()
         {
+            GenericContainer<?> container = new GenericContainer<>(IMAGE_NAME);
+
+            container.withEnv("USER", "theuser")
+                    .withEnv("PASS", "foobar")
+                    .withEnv("MGMT_USER", "admin")
+                    .withEnv("MGMT_PASS", "admin")
+                    //.withEnv("CONFIG_PATH", "/user-config/config.yaml")
```

Comment:
If this isn't being used, then delete the line.

Reply:
Still a draft and working on it
I was just testing it was working with ci but yes I will do it